web/src/beta/features/Visualizer/shared/hooks/useFieldComponent.tsx

```diff
@@ -216,6 +220,21 @@ export const FieldComponent = ({
           field?.type
         )}
       />
+    ) : field?.ui === "dynamicPropertySelector" ? (
+      <InputSelectField
+        key={field.id}
+        title={field?.title}
+        description={field?.description}
+        placeholder={field?.placeholder}
+        sketchCustomProperties={sketchCustomProperties}
```

Comment:
```suggestion
        options={sketchCustomProperties}
```
Field should be generic.